service/core/SugarWebServiceImpl.php

```diff
@@ -62,7 +62,8 @@ class SugarWebServiceImpl{
 *	     'relationship_list' -- Array - The records link field data. The example is if asked about accounts email address then return data would look like Array ( [0] => Array ( [name] => email_addresses [records] => Array ( [0] => Array ( [0] => Array ( [name] => id [value] => 3fb16797-8d90-0a94-ac12-490b63a6be67 ) [1] => Array ( [name] => email_address [value] => hr.kid.qa@example.com ) [2] => Array ( [name] => opt_out [value] => 0 ) [3] => Array ( [name] => primary_address [value] => 1 ) ) [1] => Array ( [0] => Array ( [name] => id [value] => 403f8da1-214b-6a88-9cef-490b63d43566 ) [1] => Array ( [name] => email_address [value] => kid.hr@example.name ) [2] => Array ( [name] => opt_out [value] => 0 ) [3] => Array ( [name] => primary_address [value] => 0 ) ) ) ) )
 * @exception 'SoapFault' -- The SOAP error, if any
 */
-function get_entry($session, $module_name, $id,$select_fields, $link_name_to_fields_array){
+// added parameter to make it php7 compatible
+function get_entry($session, $module_name, $id,$select_fields, $link_name_to_fields_array, $php7_compatible_parameter){
```

Comment:
I agree about naming the added function parameters after the variable names the callers are passing in.  This must be defined somewhere in the interface spec.  One reason is because when you're writing code that calls these functions in your IDE, it will give you a hint for the parameter, using the variable name as defined in the function definition.